iroha/src/lib.rs

```diff
@@ -91,7 +84,7 @@ pub struct Iroha {
     kura_blocks_receiver: CommittedBlockReceiver,
     sumeragi_message_receiver: SumeragiMessageReceiver,
     block_sync_message_receiver: BlockSyncMessageReceiver,
-    world_state_view: Arc<WorldStateView>,
+    wsv: Arc<WorldStateView>,
```

Comment:
In many places here you rename `world_state_view` into `wsv`. I don't see a need for this, and we have a rule to use full words in our contributing guide :)

Reply:
Btw, we have a file named like that.

`"wold_state_view".len()` is 16 symbols, while wsv is just 3. I would better spend my time reading and writing other things, that reading/writing `world_state_view`.

Reply:
I would personally doubt this rule from styleguide.

Generaly the good practice is that context (the name of variables, functions, ...) should depend of the scope of the thing. It should be fine to use not a full word for naming local variable in function for 5 loc. Should we rename cases like this (look at `id` variable) to a full name:
```rust
pub const fn new_big_quantity(id: DefinitionId) -> Self {
    AssetDefinition::new(id, AssetValueType::BigQuantity)
}
```

For me it is fine though to name global things (like constants, types, functions) with more context.

Rust is highly typed language (not a python/ruby or something else). Types add lots of context in terms of local variables. Most of our codebase looks like repetition of type names in variables.

I would propose to remove this rule from style guide.

References:
1. https://www.lysator.liu.se/c/pikestyle.html C style guide from go developer (No difference in language here) paragraph variable names
2. Generally rust compiler and cargo teams are fine with using variables with 1-3 character names. I don't think that their codebase is unreadable or unmaintainable. (though didn't find naming conventions)
2-1. https://github.com/rust-lang/rust/blob/master/rustfmt.toml
2-2. https://rustc-dev-guide.rust-lang.org/conventions.html
2-3. https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md

Reply:
It is kind of a holywar topic but I don't think we should change our policies now, I think this far in this project it is better to stick to the things that are already used. Though we might discuss it on status.

Reply:
We came to conclusion that we will stick to our style guide in terms of using full words, though using well known acronyms is ok. We might use `wsv` in our code as it is an acronym, and we should also rely on context aware naming.